<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="2">
     <widget class="QLineEdit" name="EditStore"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="EditPort"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>视频录制保存地址</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="BtnPicAdrs">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="EditVdAdrs"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="EditIP"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>服务器ip地址</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="EditPicAdrs"/>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="BtnVdAdrs">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>本地系统存储大小</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>车牌图片保存地址</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LabPort">
      <property name="text">
       <string>服务器端口号</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>       GB</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>智慧停车系统设置界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnReturn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
